src/components/notifications/ids.js

```diff
@@ -35,6 +35,7 @@ export default {
     REQUEST_HISTORY_DLG: "requestHistoryDlg",
     SET_PRIVILEGE_DLG: "setPrivilegeDlg",
     STATUS: "requestStatusColumn",
+    SIGN_IN_ERR_DIALOGUE: "singInErrorDialogue",
```

Comment:
😄 This is absolutely trivial, but when talking about the UI component it's `dialog` (no `ue` at the end)

Reply:
Thank you for pointing out. There is a typo in the id as well 😅